pyccel/codegen/printing/pycode.py

```diff
@@ -1136,6 +1135,12 @@ def _print_TypingFinal(self, expr):
         annotation = self._print(expr.arg)
         return f'const {annotation}'
 
+    def _print_ListPop(self, expr):
+        args = ','.join(self._print(a) for a in expr.args[1:])
+        name = self._print(expr.args[0])
```

Comment:
It would be nice to give expr.args[0] a more explicit names. This can be done using the @property decorator with class methods. ` expr.args[1:]` could stay as is if you can tell me what all these arguments are for

Reply:
**expr.args[1:]** The purpose of this is to print multiple arguments.

Reply:
Please give me an example where `pop` needs to handle multiple arguments

Reply:
Pyccel only works properly if your code runs properly in Python. `.pop(1,3,4)` will raise an error when you run your code in Python so we can safely ignore this case in Pyccel.